'SECURITY_ERR' in FileError.prototype
